docker-compose.dev.yml

```diff
@@ -18,4 +26,4 @@ services:
       - "80:80"
     depends_on:
       - backend
-    restart: always
+    restart: always
```

Comment:
Unintentional change? Looks like you removed the newline at the end of the file.